test/show.jl

```diff
@@ -65,151 +64,116 @@ end
-    │ 2   │ 10000002 │ 10000027 │ 10000052 │
-    │ 3   │ 10000003 │ 10000028 │ 10000053 │
-    │ 4   │ 10000004 │ 10000029 │ 10000054 │
-    │ 5   │ 10000005 │ 10000030 │ 10000055 │
-    │ 6   │ 10000006 │ 10000031 │ 10000056 │
-    │ 7   │ 10000007 │ 10000032 │ 10000057 │
-    │ 8   │ 10000008 │ 10000033 │ 10000058 │
-    │ 9   │ 10000009 │ 10000034 │ 10000059 │
-    │ 10  │ 10000010 │ 10000035 │ 10000060 │
-    │ 11  │ 10000011 │ 10000036 │ 10000061 │
-    │ 12  │ 10000012 │ 10000037 │ 10000062 │
-    │ 13  │ 10000013 │ 10000038 │ 10000063 │
-    │ 14  │ 10000014 │ 10000039 │ 10000064 │
-    │ 15  │ 10000015 │ 10000040 │ 10000065 │
-    │ 16  │ 10000016 │ 10000041 │ 10000066 │
-    │ 17  │ 10000017 │ 10000042 │ 10000067 │
-    │ 18  │ 10000018 │ 10000043 │ 10000068 │
-    │ 19  │ 10000019 │ 10000044 │ 10000069 │
-    │ 20  │ 10000020 │ 10000045 │ 10000070 │
-    │ 21  │ 10000021 │ 10000046 │ 10000071 │
-    │ 22  │ 10000022 │ 10000047 │ 10000072 │
-    │ 23  │ 10000023 │ 10000048 │ 10000073 │
-    │ 24  │ 10000024 │ 10000049 │ 10000074 │
-    │ 25  │ 10000025 │ 10000050 │ 10000075 │"""
+        25×5 DataFrame
+         Row │ x1        x2        x3        x ⋯
+             │ Int64     Int64     Int64     I ⋯
+        ─────┼──────────────────────────────────
+           1 │ 10000001  10000026  10000051  1 ⋯
+           2 │ 10000002  10000027  10000052  1
+           3 │ 10000003  10000028  10000053  1
+           4 │ 10000004  10000029  10000054  1
+           5 │ 10000005  10000030  10000055  1 ⋯
+           6 │ 10000006  10000031  10000056  1
+           7 │ 10000007  10000032  10000057  1
+           8 │ 10000008  10000033  10000058  1
+           9 │ 10000009  10000034  10000059  1 ⋯
+          10 │ 10000010  10000035  10000060  1
+          11 │ 10000011  10000036  10000061  1
+          12 │ 10000012  10000037  10000062  1
+          13 │ 10000013  10000038  10000063  1 ⋯
+          14 │ 10000014  10000039  10000064  1
+          15 │ 10000015  10000040  10000065  1
+          16 │ 10000016  10000041  10000066  1
+          17 │ 10000017  10000042  10000067  1 ⋯
+          18 │ 10000018  10000043  10000068  1
+          19 │ 10000019  10000044  10000069  1
+          20 │ 10000020  10000045  10000070  1
+          21 │ 10000021  10000046  10000071  1 ⋯
+          22 │ 10000022  10000047  10000072  1
+          23 │ 10000023  10000048  10000073  1
+          24 │ 10000024  10000049  10000074  1
+          25 │ 10000025  10000050  10000075  1 ⋯
+                               2 columns omitted"""
 end
 
 @testset "IOContext parameters test" begin
     df = DataFrame(A = Int64[1:4;], B = ["x\"", "∀ε>0: x+ε>x", "z\$", "A\nC"],
                    C = Float32[1.0, 2.0, 3.0, 4.0])
-    str1, size = capture_stdout() do
+    str1 = capture_stdout() do
         show(df)
     end
-    io = IOContext(IOBuffer(), :limit=>true, :displaysize=>size)
+    io = IOContext(IOBuffer(), :limit=>true, :displaysize=>(24,80))
```

Comment:
I don't understand this chance. Does that mean that `show(df)` above doesn't take into account the display size of `stdout` (which isn't always `(24,80)`)?

Reply:
Nice catch! This is a leftover due to that previous problem related to `displaysize` that is fixed in PrettyTables.jl. I will update.